src/components/Avatar/Avatar.scss

```diff
@@ -19,6 +19,10 @@ $large-size: rem(60px);
   }
 }
 
+.hidden {
+  visibility: hidden;
```

Comment:
Don't we have a mixin for visually hiding an element?

Reply:
We do it does more than this and  I'm not sure it has value here because we're not wanting if to be accessible if the image is broken.